shared-src/docs-tech-guide.rst

```diff
@@ -682,32 +682,38 @@ Write and edit files in your favorite code editor.
 
 .. _spell-check:
 
-Spell check your work
-~~~~~~~~~~~~~~~~~~~~~~
+Check your spelling
+~~~~~~~~~~~~~~~~~~~
 
 After making changes to the docs,
-you need to run the spell checker.
-To run the spell checker:
+run the spelling checker with:
+
+.. code-block:: console
+
+  $ sphinx-build -b spelling odk1-src build/spelling
```

Comment:
Hi @ping, thanks for the PR.

This command produces spelling errors since the `spelling_wordlist.txt` file is not present in `odk1-src`. You can update the command to `make odk1-spell check` which will run all the commands needed to copy the required files before running the spell checker.